src/modules/computerVision/imageSelector/ImageSelector.tsx

```diff
@@ -34,29 +34,55 @@ const ALL_IMGS: { [name: string]: any } = {
 
 export interface ImageSelectorProps {
   currImg: string;
+  currImgUrl: string;
   onSelect: (img: string, imgUrl: string) => any;
 }
 
-const ImageSelector = ({ currImg, onSelect }: ImageSelectorProps) => {
+const ImageSelector = ({
+  currImg,
+  currImgUrl,
+  onSelect,
+}: ImageSelectorProps) => {
+  const [dimensions, setDimensions] = useState({ width: 0, height: 0 });
+  const imgRef = useRef<HTMLImageElement>(null);
+
   useEffect(() => onSelect(currImg, ALL_IMGS[currImg]), []);
 
   const makeImg = (key: string) => (
     // eslint-disable-next-line
     <img
       key={key}
-      className={key === currImg ? 'selected' : ''}
+      className={`img-visible ${key === currImg ? 'selected' : 'unselected'}`}
       src={ALL_IMGS[key]}
       onClick={() => onSelect(key, ALL_IMGS[key])}
     />
   );
 
   return (
     <div className="image-selector">
+      {/* eslint-disable-next-line */}
+      <img
+        key={`hidden ${currImg}`}
+        ref={imgRef}
+        className="hidden"
+        src={currImgUrl}
+        onLoad={() =>
+          setDimensions({
+            width: imgRef.current?.width ?? 0,
+            height: imgRef.current?.height ?? 0,
+          })
+        }
+      />
       Select Image
       <div className="selection-window">
         {Object.keys(ALL_IMGS).map(key => ALL_IMGS[key] && makeImg(key))}
       </div>
-      <ImageUploader onSelect={onSelect} />
+      <ImageUploader currImg={currImg} onSelect={onSelect} />
+      Image Size:&nbsp;
```

Comment:
kinda pedantic but can you separate the "Image Size" into its own `p` tag and "width px by height px" into another `p` tag, then have both of those in a responsive `div` (ie with the className "flex flex-col md:flex-row") so that the "width px by height px" moves onto a separate line on mobile